src/main/java/domain/Hand.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Hand {
+    private static final int BLACKJACK_CONDITION = 21;
+
+    private final List<Card> cards;
+
+    public Hand(final Card... cards) {
+        this(List.of(cards));
+    }
+
+    public Hand(final List<Card> cards) {
+        this.cards = new ArrayList<>(cards);
+    }
+
+    public Score score() {
+        Score score = new Score(0);
```

Comment:
자주 사용될 의미있는 값은 미리 생성해 두는 것도 좋을 수 있겠네요

```suggestion
        Score score = Score.ZERO;
```